### DISCUSSION TITLE
podofo package needs package source change from sourceforge to github

### DISCUSSION THREAD

**Discussion by luzpaz** [Homebrew/discussions#4622] [Everyday usage] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
m/a
```

### Description of issue

Per the sourceforge URL (https://sourceforge.net/projects/podofo/)  

> Development of PoDoFo has been moved to GitHub: https://github.com/podofo/podofo

Current homebrew formula version for podofo is 0.9.8 and pulling from sourceforge

https://github.com/Homebrew/homebrew-core/blob/aec3b08de10e5b74b1bdced322858b908f524ec1/Formula/podofo.rb#L4

The actual current podofo stable is v0.10.1 (https://github.com/podofo/podofo/releases/tag/0.10.1)
 
Would someone mind updating the formula to reflect that change ?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by luzpaz** [comment]
I don't own a mac so i can't run diagnostics on it.

**Reply by SMillerDev** [reply, marked as answer]
That's what we have CI and Linux/Docker support for

**Reply by osalbahr** [reply]
Try https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#formulae-related-pull-request. The steps don't require being on a macOS machine.

**Reply by nandahkrishna** [reply]
I opened https://github.com/Homebrew/homebrew-core/pull/135848 to update the formula. Thanks @luzpaz.
